homeassistant/components/august/gateway.py

```diff
@@ -32,29 +38,14 @@ def __init__(self, hass):
         self._access_token_cache_file = None
         self._hass = hass
         self._config = None
-        self._api = None
-        self._authenticator = None
-        self._authentication = None
-
-    @property
-    def authenticator(self):
-        """August authentication object from py-august."""
-        return self._authenticator
-
-    @property
-    def authentication(self):
-        """August authentication object from py-august."""
-        return self._authentication
+        self.api = None
+        self.authenticator = None
+        self.authentication = None
 
     @property
     def access_token(self):
         """Access token for the api."""
-        return self._authentication.access_token
-
-    @property
-    def api(self):
-        """August api object from py-august."""
-        return self._api
+        return self.authentication.access_token
```

Comment:
Is it safe to not have a guard here if `self.authentication` is None?

Reply:
Yes. Its working that way now.